src/views/Resources/Resources.vue

```diff
@@ -4,7 +4,8 @@
     class="text-content"
   >
     <h2><span class="lowlight">Further Research</span></h2>
-    <ul>
+    <p>For more information about USGS work on wildfires and water supplies, visit the <a href="https://www.usgs.gov/special-topic/fire">Wildland Fire Science</a>, the <a href="https://ca.water.usgs.gov/wildfires/wildfires-water-quality.html">California Water Science Center,</a> or any of the USGS peer-reviewed research on wildfires, some of which is listed below.</p>
```

Comment:
Small thing here, reads
>For more information about USGS work on wildfires and water supplies, visit the Wildland Fire Science, the California Water Science Center, or any of the USGS peer-reviewed research on wildfires, some of which is listed below.
That first 'the' before 'Wildland' feels weird. Maybe say 'consult Wildland Fire Science' or 'consult the Wildland Fire Science page'?